AUTHORS.md

```diff
@@ -0,0 +1,52 @@
+# Credits
+
+---
+© 2024 **xDEM developers**.
+
+**xDEM** is licensed under permissive Apache 2 license (See LICENSE file).
+
+All contributors listed in this document are part of the **xDEM developers**, and their
+contributions are subject to the project's copyright under the terms of the
+[Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+
+See CONTRIBUTING.md for more details on Contributor License Agreement.
+
+This file keeps track of authors contributions.
+
+## Development Lead
+
+---
+
+- **Romain Hugonnet** [@rhugonnet](https://github.com/rhugonnet) <romain.hugonnet@gmail.com>
+- **Amaury Dehecq** [@adehecq](https://github/adehecq) <amaury.dehecq@univ-grenoble-alpes.fr>
+- **Erik Schytt Mannerfelt** [@erikmannerfelt](https://github/erikmannerfelt)
+
+## Contributors
+
+---
+
+- **Friedrich Knuth** [@friedrichknuth](https://github/friedrichknuth)
+- **Andrew Tedstone** [@atedstone](https://github/atedstone)
+- **Zhihao LIU** [@liuh886](https://github/liuh886)
+- **Diego Cusicanqui** [@cusicand](https://github/cusicand)
+- **Alessandro Gentilini** [@alessandro-gentilini](https://github/alessandro-gentilini)
+- **Ferdinand Schenck** [@fnands](https://github/fnands)
+- **Johannes Landmann** [@jlandmann](https://github/jlandmann)
+- **Valentin Schaffner** [@vschaffn](https://github/vschaffn) <valentin.schaffner@cs-soprasteria.com>
+- **Alice De Bardonnèche-Richard** [@adebardo](https://github/adebardo) <alice.de-bardonneche-richard@cs-soprasteria.com>
+- **Emmanuel Dubois** [@duboise-cnes](https://github/duboise-cnes) <emmanuel.dubois@cnes.fr>
+- **Bob McNabb** [@iamdonovan](https://github/iamdonovan)
+- **Enrico Mattea** [@MatteaE](https://github.com/MatteaE)
+- **Amelie Froessl** [@ameliefroessl](https://github.com/ameliefroessl)
+- **Simon Gascoin** [@sgascoin](https://github.com/sgascoin)
+
+Update here with new contributors.
+
+
+## Original Developers/Designers/Supporters
+
+---
+
+- **Romain Hugonnet** [@rhugonnet](https://github.com/rhugonnet) <romain.hugonnet@gmail.com>
+- **Amaury Dehecq** [@adehecq](https://github/adehecq) <amaury.dehecq@univ-grenoble-alpes.fr>
+- **Erik Schytt Mannerfelt** [@erikmannerfelt](https://github/erikmannerfelt)
```

Comment:
@erikmannerfelt Hello, I’m tagging you because we are about to merge this PR that will change the license of xDEM. Would you like us to include your email?